src/pykeen/losses.py

```diff
@@ -45,8 +45,8 @@ class SetwiseLoss(Loss):
     """Setwise loss functions compare the scores of several triples."""
 
 
-class BCELoss(PointwiseLoss, nn.BCELoss):
-    r"""A wrapper around the PyTorch binary cross entropy loss.
+class BCEWithLogitsLoss(PointwiseLoss, nn.BCEWithLogitsLoss):
```

Comment:
can't we just leave the inheriting class as BCELoss?

Reply:
otherwise this PR should also improve the lookup to allow for synonyms (in case people write just BCE it should correspond to this one)

Reply:
Because BCE and BCWithLogitLoss are two different loss functions in PyTorch, I would not use BCE as a synonym since it might confuse.
I think it is better that the user is aware of which explicit loss function he/she is using.

Reply:
Fair enough! I would like to leave the infrastructure for synonyms though. I think one of the loss functions we implemented also is called the hinge loss, but I always forget. Is that right? Which one is it?

Reply:
We have the pointwise [hinge loss](https://en.wikipedia.org/wiki/Hinge_loss), and the pairwise hinge loss which is usally callede margin ranking loss.

Reply:
Thanks, added in f1d4ebe1

Reply:
Thanks @cthoyt!